### ISSUE TITLE
Error message: gitflow-common: line 79: [: -eq: unary operator expected

### ISSUE THREAD

**Issue by andymwood** [petervanderdoes/gitflow-avh#276]
I'm getting the above error message from git-flow after executing the following sequence of commands:

```
mkdir project
cd project
git flow init
git flow feature start f
touch a
git add a
git comit -m comment
git flow feature finish f
```

Output form the last command is:

> flags:WARN flag short name (p) already defined
> Switched to branch 'develop'
> Merge made by the 'recursive' strategy.
>  a | 0
>  1 file changed, 0 insertions(+), 0 deletions(-)
>  create mode 100644 a
> Deleted branch feature/f (was 9c59d5b).
> /home/fge/git/2.9.3/bin/gitflow-common: line 79: [: -eq: unary operator expected
> 
> Summary of actions:
> - The feature branch 'feature/f' was merged into 'develop'
> - Feature branch 'feature/f' has been locally deleted
> - You are now on branch 'develop'

This is with the latest version - 1.10.0, running under openSUSE 13.2.
The error does not occur with the version of git-flow installed from the openSUSE repositories (1.9.1)

**Comment by sgtsquiggs** [comment]
Getting this exact error myself:

```
> git flow version
1.10.0 (AVH Edition)
> git --version
git version 2.9.3
```

**Comment by petervanderdoes** [comment]
What is the prefix you use for the feature branch?

**Comment by andymwood** [comment]
> What is the prefix you use for the feature branch?

'feature/', the default.

**Comment by petervanderdoes** [comment]
Fix per version 1.10.1 Thanks for reporting
